### PR TITLE
chore: Add pr as condition

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### CODE REVIEW

**Comment by kerm1it** [comment]
没有用到密钥直接用 `pull_request` 就好了吧？

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: ✅ test
 
-on: push
+on:
```
Comment (kerm1it) [inline .github/workflows/test.yml:3]:
```suggestion
on: [push,pull_request]
```

Comment (zombieJ) [reply .github/workflows/test.yml:3]: 👌
这里需要 `pull_request_target` 使用 target 的上下文。

**Comment by zombieJ** [comment]
有 `secrets.BUNDLESIZE_GITHUB_TOKEN`

**Pull request merged**